### DISCUSSION TITLE
Unable to enable Legacy Camera Support [Command Not Found]

### DISCUSSION THREAD

**Discussion by neocody** [ptrumpis/snap-camera-server#141] [Q&A]
everyone,

I'm running into an issue on my Mac running macOS 14.4.1. I've been trying to enable legacy camera support as per [Apple's instructions](https://support.apple.com/en-us/108387), but every time I enter the command, it returns a "command not found" error. I posted about this in the other thread on the matter but it's almost been a month and nobody replied, so I'm creating this thread. [[Screenshot](https://imgur.com/a/jxGkbes)]

Through my extensive (it was) I found other users who encountered the problem had to disable FileVault in order for it to work and needed to ensure the APFS volume was mounted. In neither scenario did the command actually execute.

Has anyone else encountered this issue? Any advice or solutions would be greatly appreciated. Thanks in advance!

**Comment by mathijsstegink** [comment]
I have the same problem! Would be great if someone has some suggestions!

**Comment by ptrumpis** [comment]
You probably need to disable FileVault first.
[Turn off FileVault on Mac](https://support.apple.com/guide/mac-help/mchlp2560/14.0/mac/14.0)

**Comment by neocody** [comment]
It is off, that is the basis for this post because disabling filevault did not solve the problem.

**Reply by ptrumpis** [reply]
Sorry I see, I got things twisted with another post.

**Comment by ptrumpis** [comment]
Does this fix your problem by any chance?
```sh
diskutil list | awk '/disk[0-9]s[0-9]/ {print $NF}' | while read disk; do diskutil mount "/dev/$disk" && mount -uw "/Volumes/$(diskutil info /dev/$disk | awk -F': ' '/Mount Point/ {print $2}')"; done; system-override legacy-camera-plugins-without-sw-camera-indication=on
```

**Comment by neocody** [comment]
I just bit the bullet and bought "FilterOnMe" app. The subscription price is highway robbery but I'll put a task to try this command before my subscription renews to try this then
